tensorflow/core/kernels/conv_ops_impl.h

```diff
@@ -90,6 +90,41 @@ namespace tensorflow {
 typedef Eigen::ThreadPoolDevice CPUDevice;
 typedef Eigen::GpuDevice GPUDevice;
 
+// Maximum tensor size (in bytes) that cuDNN can handle safely.
```

Comment:
This doesn't belong here.

----

Please make sure all of your current PRs are cleaned of AI slop before asking for more reviews. Next PR review of AI slop will get all of your PRs desk-rejected. Please stop wasting reviewer's time!